pkg/bot/cloudslack.go

```diff
@@ -32,8 +34,11 @@ import (
 )
 
 const (
-	APIKeyContextKey       = "X-Api-Key"       // #nosec
-	DeploymentIDContextKey = "X-Deployment-Id" // #nosec
+	APIKeyContextKey        = "X-Api-Key"       // #nosec
+	DeploymentIDContextKey  = "X-Deployment-Id" // #nosec
+	retryDelay              = time.Second
+	maxRetries              = 25
+	successIntervalDuration = 5 * time.Minute
```

Comment:
WDYT about 1 or 2 minutes?

Reply:
discussed f2f, randomly picked 3 😄  for now we don't have enough data to come up with well explained defaults